SCALERS.md

```diff
@@ -0,0 +1,57 @@
+# Scaler Governance
+
+This document describes how KEDA handles governance of built-in scalers and how it relates to external scalers maintained by the community.
+
+## Scalers in KEDA
+
+Each event source has unique characteristics on how KEDA needs to connect to it and the types of metrics available to drive scaling. KEDA is made up of many *scalers*, or simple extensions to the core KEDA operator that allows it to connect to an event source.
+
+> 📝 You can find a full list of supported scalers in [our documentation](https://keda.sh/docs/latest/scalers/).
+
+Scalers are shipped with KEDA in two ways:
+
+- **Built-in Scalers** - Scalers are packaged and shipped as part of the core KEDA runtime
+- **External Scalers** - Scalers that are running outside of KEDA but are capable of being used to scale workloads with. Workloads can be scaled by using the "External" or "External Push" scalers which integrate with the external scalers by using gRPC to drive scale.
+
+### Choosing the right model for a scaler
+
+When introducing a new scaler, it is important to choose the right model for it - Should it be a built-in scaler or an external scaler?
+
+The general rules of thumb for built-in scalers are the following:
+
+- They must live in the [core KEDA repository](http://github.com/kedacore/keda)
+- It follows the [KEDA core release cycle](https://github.com/kedacore/keda/blob/main/ROADMAP.md#upcoming-release-cycles)
+- It has no additional infrastructure requirements
+- It is general-purpose and not specific to a single end-user
+- It is written in Go
+
+If all of the above are true, then it is a potential case for a new built-in scaler. We recommend opening a feature request to propose and discuss it before doing the actual implementation though. This is to avoid throwing things away if it does not meet the needs after all.
+
+### Requirements for a built-in scaler
+
+In order to contribute a scaler to KEDA's core, it must meet the following requirements:
+
+- A scaler MUST be written in Go
+- A scaler maintainer(s) MUST be assigned (individual(s)/company) and will be tagged for bug reports
+- A scaler MUST have e2e and unit tests to guarantee its quality
+- A scaler MUST be backwards compatible until the next KEDA major version
```

Comment:
I would omit this. Next Major version of KEDA is something, that I'd like to avoid in near future.
I think it is safe to deprecate some fields of a scaler, if we have some deprecation period, for example 2 releases, where we support both new and old functionality, where possible.

Reply:
Then we should come up with a version for scalers as discussed in the main repo along with a deprecation policy, but that's fine for me!

Reply:
```suggestion
```